utils/node_local_test.py

```diff
@@ -2805,7 +2800,7 @@ def test_cont_copy(self):
                '--src',
                src_dir.name,
                '--dst',
-               'daos://{}/{}'.format(self.pool.uuid, self.container)]
+               f'daos://{self.pool.uuid}/{self.container}']
```

Comment:
Side note - If you ever get the chance, we should switch from pool UUID to label. Mohamad suggested before that the datamover tools support labels only to be more user friendly and simplify logic

Reply:
Good point.  There's another pylint issue in this file that the Pool class has a id() method whose name is invalid however it's also widely used so changing it would expand the number of changes here.

Best would be to change the Class to only support labels and then self.pool could be used throughout, then the question is would it be better to make code changes and update all the invocation sites in this PR or separately?  I can do either.